core/src/main/scala/io/finch/ToResponse.scala

```diff
@@ -124,14 +124,29 @@ object ToResponse extends HighPriorityToResponseInstances {
     }
   }
 
+  implicit def coproductToResponse[C <: Coproduct, CT <: String](
+    implicit ctr: CoproductToResponse.Aux[C, CT]
+  ): Aux[C, CT] = ctr
+}
+
+trait CoproductToResponse[C <: Coproduct] extends ToResponse[C]
```

Comment:
Just an idea. How do you feel about moving this new typeclass under the `ToResponse` companion object? This way we can reduce the top-level/user-facing API surface. I'd imagine something like `ToResponse.FromCoproduct` should do it:

```scala
object ToReponse {
  type Aux = ...
  trait FromCoprodct extends ToResponse[...]
}

```

Reply:
Fine by me, I was wondering the same thing :)

Reply:
Unless you already working on it, I'd be happy to see this merged in its current state and then addressing this comment later. Whatever you prefer - just le me know.